runtime/jcl/common/jclcinit.c

```diff
@@ -124,6 +124,13 @@ jint computeFullVersionString(J9JavaVM* vm)
 			j2se_version_info = "13.?";
 		}
 		break;
+	case J2SE_V14:
```

Comment:
Suggested to Keith this code can be modified to use JAVA_SPEC_VERSION_STRING so it doesn't need to be updated every 6 months.

Reply:
Done.